tests/backup/backup_helper.go

```diff
@@ -995,10 +996,38 @@ func CleanupCloudSettingsAndClusters(backupLocationMap map[string]string, credNa
 			Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cloud cred %s", credName))
 		}
 	}
-	err := DeleteCluster(SourceClusterName, orgID, ctx)
-	Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cluster %s", SourceClusterName))
-	err = DeleteCluster(destinationClusterName, orgID, ctx)
-	Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cluster %s", destinationClusterName))
+
+	log.Infof("Deleting the application cluster and their respective cloud credentials if present")
+	kubeconfigs := os.Getenv("KUBECONFIGS")
+	Inst().Dash.VerifyFatal(kubeconfigs != "", true, "Getting KUBECONFIGS Environment variable")
```

Comment:
How about?

kubeconfigs != "" | len(strings.Split(kubeconfigs, ",")) < 2

Reply:
Our run will fail when we have purposely given more than 2 kubeconfigs.
CreateApplicationClusters has also been changed after discussion to accomate future requirement of more than 2 kubeconfigs

Reply:
My bad. I meant kubeconfigs != "" | len(strings.Split(kubeconfigs, ",")) < 2

Reply:
done